docs/sql/float.md

```diff
@@ -89,14 +89,50 @@ REAL '1.23'  -- string style
   </tr>
   <tr>
     <td><code>SIN(value)</code></td>
-    <td>The sine of value as radians. <code>sin</code> only supports argument of type double, so all other types are cast to double. Returns a double.</td>
+    <td>The sine of value, returned in radians. <code>sin</code> only supports argument of type double, so all other types are cast to double. Returns a double.</td>
   </tr>
   <tr>
     <td><code>COS(value)</code></td>
-    <td>The cosine of value as radians. <code>cos</code> only supports argument of type double, so all other types are cast to double. Returns a double.</td>
+    <td>The cosine of value, returned in radians. <code>cos</code> only supports argument of type double, so all other types are cast to double. Returns a double.</td>
   </tr>
   <tr>
     <td><code>PI</code></td>
     <td>Returns the approximate value of <code>PI</code> as double. Note that <code>()</code> is not required. Example: <code>SELECT PI;</code></td>
```

Comment:
is not required or not allowed?
These are different.
In standard SQL functions with no arguments should use no (), but I don't remember what we do. This is configurable in Calcite. I think we do allow them, and then this comment is correct.

Reply:
Right now `()` is not allowed.